src/dtx/dtx_srv.c

```diff
@@ -412,6 +412,14 @@ dtx_init(void)
 
 	D_INFO("Set DTX RPC helper threshold as %u\n", dtx_rpc_helper_thd);
 
+	str = getenv("DTX_BATCHED_ULT_MAX");
+	if (str != NULL)
+		dtx_batched_ult_max = atoi(str);
+	else
+		dtx_batched_ult_max = DTX_BATCHED_ULT_DEF;
```

Comment:
d_getenv_int("DTX_BATCHED_ULT_MAX", &dtx_batched_ult_max) is supposed to get integer env var, the dtx_batched_ult_max should be initialized as DTX_BATCHED_ULT_DEF above.

BTW, if the env var is exported to user, it should be named "DAOS_DTX_XXX" and mentioned in the doc.